config/default.yaml

```diff
@@ -43,8 +43,10 @@ cards:
     name: Planetary Bombers
     description: When the planet seems lost, the Planetary Bombers will destroy everything on it
     special: true
+    defense: true
+    planet: true
   tankBattalion:
-    name: Tank Battalioon
```

Comment:
I don't know, I think a Tank Battalioon sounds exciting.  Tanks in hot air balloons?